script/tool/lib/src/build_examples_command.dart

```diff
@@ -310,13 +329,29 @@ class BuildExamplesCommand extends PackageLoopingCommand {
   }) async {
     final String enableExperiment = getStringArg(kEnableExperiment);
 
+    final _ExtraConfiguration extraBuildFlagsConfiguration =
+        _readExtraBuildFlagsConfiguration(example.directory);
+
+    if (extraBuildFlagsConfiguration.configs.isNotEmpty) {
+      for (final String config in extraBuildFlagsConfiguration.configs) {
+        await processRunner.runAndStream(
+          flutterCommand,
+          <String>[
```

Comment:
`flutter config` is a global command; individual packages should not be opting the entire machine into config flags as this will cause unexpected side-effects for all other tests on the same bot. Please revert this tool change.

What is the goal of building this into the tool instead of using `pubspec.yaml`?

Reply:
I didn't know that was possible! That makes things much simpler of course.